src/main/java/com/whalespottingjava/services/SightingService.java

```diff
@@ -32,6 +34,12 @@ public List<Sighting> getAllApprovedSightings() { // TODO: 12/10/2023 Returns ra
         return this.sightingRepository.findByApprovedTrue();
     }
 
+    public void addBulkSightings(List<Sighting> sightings) {
+        // Validate sightings (Use jakarta?)
```

Comment:
is this validation now done? I.e. can we remove the comment?

Reply:
The validation is not done yet, there is a ticket to track the issue, it's my next job after this merge. [Ticket.](https://github.com/users/charlottemothersole/projects/1/views/1?pane=issue&itemId=41634091)